modules/custom/az_accordion/js/az-accordion-formatter.es6.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file
+ * Provides scroll to and expand functionality for accordion based on anchor links.
+ */
+
+((window, document) => {
+  function init() {
+    // Function to handle accordion based on hash
+    const handleAccordion = (hash) => {
+      if (hash) {
+        const $targetAccordion = document.querySelector(hash);
+        if (
+          $targetAccordion &&
+          hash !== '#' &&
+          hash.startsWith('#accordion-') &&
+          $targetAccordion.classList.contains('collapse')
+        ) {
+          const yOffset = -10;
```

Comment:
We can change the offset here:
```suggestion
          const yOffset = -20;
```

Or find the size of the collapse button height and put it here